### DISCUSSION TITLE
Função CTRL + C / CTRL + V

### DISCUSSION THREAD

**Discussion by brasilsolucoes** [rustdesk/rustdesk#4894] [Feature Request]
Função extremamente útil, atualmente pecisa abrir o transferidor de arquivos e ir abrindo as pastas para realizar a passagem d eum arquivo de um pc pro outro

**Comment by remoover** [comment]
this will be great to add it .. same way working teamviewer...

**Reply by BayJay** [reply]
File transfer (ctrl+width ctrl+v) between computers is very necessary
